rosdep/python.yaml

```diff
@@ -5151,6 +5151,11 @@ python3-dev:
   openembedded: [python3@openembedded-core]
   rhel: ['python%{python3_pkgversion}-devel']
   ubuntu: [python3-dev]
+python3-distutils:
+  debian: [python3-distutils]
+  fedora: [python3]
+  gentoo: [dev-lang/python]
+  ubuntu: [python3-distutils]
```

Comment:
Similarly, this is only supported on bionic and later, but we still support xenial for kinetic.  Please use the null syntax here to disable the xenial support.

Reply:
null syntax for xenial in fe9bde3